include/RandLAPACK/drivers/rsvd.hh

```diff
@@ -1,9 +1,8 @@
-#ifndef BLAS_HH
-#include <blas.hh>
-#define BLAS_HH
-#endif
+#ifndef randlapack_drivers_rsvd_h
+#define randlapack_drivers_rsvd_h
 
 #include "../comps/qb.hh"
+#include "blaspp.h"
```

Comment:
I'm confused about these two lines. The first line seems to imply that include statements with quotations are relative to the position of the current file. But if that's true, then the second include statement should be ``#include "../blaspp.h``. Is the current format really valid? If so, can you explain how these statements are being parsed?

Separately, I don't seriously want to support Windows, but is it okay to specify file paths using forward slashes when using C++ compilers on Windows machines?